check_springboot_actuator.py

```diff
@@ -113,12 +121,12 @@ def handle_version_2():
                 http_status_counter[status] = (
                         http_status_counter.get(status, 0) + measurement['value'])
             else:
-                helper.add_metric(label="%s_%s" % (key.replace('.', '_'), measurement['statistic'].lower()), value=measurement['value'])
-
+                helper.add_metric(label="%s.%s" % (key, measurement['statistic'].lower()), value=measurement['value'])
```

Comment:
this might break existing graphs/integrations

Reply:
Yes.
It was not my intention to pull request all my changes at this point. I will send a new one for just the 503 return code